lte/gateway/c/oai/tasks/s1ap/s1ap_mme_nas_procedures.c

```diff
@@ -45,69 +44,83 @@
 #include "3gpp_36.413.h"
 #include "INTEGER.h"
 #include "OCTET_STRING.h"
-#include "S1AP-PDU.h"
-#include "S1ap-AllocationAndRetentionPriority.h"
-#include "S1ap-E-RABLevelQoSParameters.h"
-#include "S1ap-E-RABToBeSetupItemBearerSUReq.h"
-#include "S1ap-E-RABToBeSetupItemCtxtSUReq.h"
-#include "S1ap-EUTRAN-CGI.h"
-#include "S1ap-EncryptionAlgorithms.h"
-#include "S1ap-GBR-QosInformation.h"
-#include "S1ap-GUMMEI.h"
-#include "S1ap-IntegrityProtectionAlgorithms.h"
-#include "S1ap-NAS-PDU.h"
-#include "S1ap-PLMNidentity.h"
-#include "S1ap-ProcedureCode.h"
-#include "S1ap-S-TMSI.h"
-#include "S1ap-SecurityKey.h"
-#include "S1ap-TAI.h"
-#include "S1ap-TransportLayerAddress.h"
-#include "S1ap-UEAggregateMaximumBitrate.h"
-#include "S1ap-UESecurityCapabilities.h"
+#include "S1ap_S1AP-PDU.h"
+#include "S1ap_AllocationAndRetentionPriority.h"
+#include "S1ap_E-RABLevelQoSParameters.h"
+#include "S1ap_E-RABToBeSetupItemBearerSUReq.h"
+#include "S1ap_E-RABToBeSetupItemCtxtSUReq.h"
+#include "S1ap_EUTRAN-CGI.h"
+#include "S1ap_EncryptionAlgorithms.h"
+#include "S1ap_GBR-QosInformation.h"
+#include "S1ap_GUMMEI.h"
+#include "S1ap_IntegrityProtectionAlgorithms.h"
+#include "S1ap_NAS-PDU.h"
+#include "S1ap_PLMNidentity.h"
+#include "S1ap_ProcedureCode.h"
+#include "S1ap_S-TMSI.h"
+#include "S1ap_SecurityKey.h"
+#include "S1ap_TAI.h"
+#include "S1ap_TransportLayerAddress.h"
+#include "S1ap_UEAggregateMaximumBitrate.h"
+#include "S1ap_UESecurityCapabilities.h"
 #include "TrackingAreaIdentity.h"
 #include "asn_SEQUENCE_OF.h"
 #include "nas/securityDef.h"
 #include "s1ap_state.h"
-#include "S1ap-CauseMisc.h"
-#include "S1ap-CauseNas.h"
-#include "S1ap-CauseProtocol.h"
-#include "S1ap-CauseRadioNetwork.h"
-#include "S1ap-CauseTransport.h"
-#include "S1ap-E-RABItem.h"
+#include "S1ap_CauseMisc.h"
+#include "S1ap_CauseNas.h"
+#include "S1ap_CauseProtocol.h"
+#include "S1ap_CauseRadioNetwork.h"
+#include "S1ap_CauseTransport.h"
+#include "S1ap_E-RABItem.h"
```

Comment:
Why is "-" is present within S1ap_E-RABItem.h?

Reply:
These are ASN.1C files which are generated automatically .